### DISCUSSION TITLE
I have create StructuredChat Agent node, if you give me push permissions I can contribute this :)

### DISCUSSION THREAD

**Discussion by niron1** [FlowiseAI/Flowise#307] [Ideas]
I have create StructuredChat Agent node, if you give me push permissions I can contribute this :)

**Comment by HenryHengZJ** [comment]
thanks for willing to contribute! normally you'd fork flowise and work on your branch, then open a pr pointing to official flowise main branch.

Here's a step by step guide - https://github.com/FlowiseAI/Flowise/blob/main/CONTRIBUTING.md
